### ISSUE TITLE
standard deviation issue

### ISSUE THREAD

**Issue by Aimen1996** [thuml/Nonstationary_Transformers#24]
what i believe is while calculating standard deviation, there should be "x_raw" rather than "x_enc". 
std_enc = torch.sqrt(torch.var(x_enc, dim=1, keepdim=True, unbiased=False) + 1e-5).detach()
